saleor/dashboard/product/forms.py

```diff
@@ -59,6 +61,18 @@ class Meta:
                 'Product class form label',
                 'Attributes common to all variants')}
 
+    def __init__(self, *args, **kwargs):
+        super(ProductClassForm, self).__init__(*args, **kwargs)
+        if settings.VATLAYER_ACCESS_KEY is not None:
+            rate_choices = [('', '--------'), ('standard', 'standard')]
```

Comment:
Maybe it should but we can't currently translate any of the other labels in the same choice list (they come from an API).